### ISSUE TITLE
[AWS] Set CipherSuites in Kubernetes Service

### ISSUE THREAD

**Issue by billyteves** [kubernetes/kubernetes#54469]
Is there a way to set specific cipher suite to be use in Elastic Load Balancing? 
Right now, i am trying to set the cipher suite to use only the TLSv1.2 via AWS Console. 
I was wondering if there is a way to set this up via service's annotation? Thanks!

Regards,

**Comment by grodrigues3** [comment]
@kubernetes/sig-aws-misc

**Comment by georgebuckerfield** [comment]
I don't think this is possible at the moment. I'd be happy to take a look at adding this in if it's something people would find useful?

**Comment by micahhausler** [comment]
#54507 is merged, and will be a part of 1.9.

You will be able to use the annotation `service.beta.kubernetes.io/aws-load-balancer-ssl-negotiation-policy` to specify an AWS pre-defined policy. To get the list of accepted policies, run 
```
aws elb describe-load-balancer-policies --query 'PolicyDescriptions[].PolicyName'
```

If you need a custom policy, see [my comment](https://github.com/kubernetes/kubernetes/pull/54507#issuecomment-339329158) on the PR.

This can probably be closed
